en/docs/develop/intro-integration-development.md

```diff
@@ -154,7 +151,7 @@ To start developing integration solutions, you need to first <a href="{{base_pat
                         <a href="{{base_path}}/develop/customizations/creating-custom-mediators/">Custom Mediator</a>
                     </li>
                     <li>
-                        <a href="{{base_path}}/develop/customizations/creating-custom-Inbound-endpoint/">Custom Inbound Enpoint</a>
+                        <a href="{{base_path}}/develop/customizations/creating-custom-inbound-endpoint/">Custom Inbound Enpoint</a>
```

Comment:
Shall we correct this to **Custom Inbound Endpoint**

Reply:
Fixed in 81d128aef3c862585d1f0490fdac45afa86ddfb4.